snap/snapcraft.yaml

```diff
@@ -52,15 +54,31 @@ parts:
     plugin: cmake
     configflags:
     - -DCMAKE_BUILD_TYPE=Release
+    - -DLLVM_ROOT_DIR=../../llvm/install
     stage:
     - -*
     prime:
     - -*
     build-packages:
     - build-essential
-    - llvm-dev
     - libconfig++-dev
     - libedit-dev
     - zlib1g-dev
     stage-packages:
     - libconfig9
+    after:
+    - llvm
+
+  llvm:
+    source: http://releases.llvm.org/3.9.1/llvm-3.9.1.src.tar.xz
+    plugin: cmake
+    configflags:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DLLVM_BINUTILS_INCDIR=/usr/include
+    stage:
+    - -*
+    prime:
+    - -*
+    build-packages:
+    - binutils-dev
```

Comment:
I could probably build the latest `binutils` instead of using the distro package, but since all that it's needed for (AFAICT) is the `plugin-api.h` header, and the API is stable, it seems overkill.